src/encoded/commands/generate_ontology.py

```diff
@@ -1,22 +1,19 @@
-import boto3
-import datetime
-import os
-import json
-import sys
 import argparse
+import ast
+# import boto3
+# import datetime
+import json
+# import os
+import pkg_resources
 import re
 import requests
+import sys
 
 from collections import Counter
-from dateutil.relativedelta import relativedelta
-from dcicutils.ff_utils import (
-    get_authentication_with_server,
-    get_metadata,
-    search_metadata,
-    unified_authentication
-)
-from dcicutils.s3_utils import s3Utils
-from pyramid.paster import get_app
+# from dateutil.relativedelta import relativedelta
+from dcicutils.ff_utils import get_authentication_with_server, get_metadata, search_metadata, unified_authentication
+# from dcicutils.s3_utils import s3Utils
+# from pyramid.paster import get_app
```

Comment:
why are these left commented out, a linter-related reason?

Reply:
I often do a two-phased delete where if I'm uncertain about something, I comment something out to see if it passes tests and then it reminds me if there was a test failure that something was changed. If it passes, I try to come back later and remove it unless I think it might soon be re-added.

There is a use-case where things come and go and it's easier to leave a comment that you can just add/remove. I don't think these are that case.

Reply:
I'm deleting this on the next commit, not yet uploaded.